deployments/kubernetes/monitoring/prometheus/config-map.yml

```diff
@@ -127,4 +127,10 @@ data:
         - source_labels: [__param_target]
           target_label: instance
         - target_label: __address__
-          replacement: incrudibles-k8s.db.pdl.cmu.edu:32005
\ No newline at end of file
+          replacement: incrudibles-k8s.db.pdl.cmu.edu:32005
+      
+      - job_name: 'timescaledb' 
+        metrics_path: /timescaledb
+        static_configs:
+          - targets:
+              - incrudibles-testing.db.pdl.cmu.edu:30003
```

Comment:
Should this be the production endpoint?

Reply:
I agree with Dan, this should be `incrudibles-k8s.db.pdl.cmu.edu:32006`